pkg/madmin/group-commands.go

```diff
@@ -41,11 +41,11 @@ func (adm *AdminClient) UpdateGroupMembers(g GroupAddRemove) error {
 	}
 
 	reqData := requestData{
-		relPath: "/v1/update-group-members",
+		relPath: "/v2/update-group-members",
```

Comment:
We should make this as a top-level constant and then use that in admin-router.go